how-to/software/upgrade-your-release.md

```diff
@@ -22,6 +36,7 @@ To ensure a successful upgrade, review the following items:
   ```bash
   sudo apt update
   sudo apt upgrade
+  sudo reboot
```

Comment:
recommending a reboot after every upgrade is not something we should do. rather we could hint at "needrestart" and needs-reboot status after kernel upgrades

Reply:
This is not for every upgrade, but for the upgrade prior to the *release* upgrade. Still, I agree that a reboot should only be done if requested by the system. And if I'm not mistaken, `do-release-upgrade` will tell you about it if needed.
So I think replace the reboot command with a note that depending on the packages that were upgrade, you might have to reboot the system if asked to do so.